chain.go

```diff
@@ -145,13 +149,15 @@ func (c *chain) setResponse(resp *Response) {
 func (c *chain) clone() *chain {
 	copy := *c //nolint
 	copy.context.Path = append(([]string)(nil), c.context.Path...)
+	copy.context.AliasedPath = append(([]string)(nil), c.context.AliasedPath...)
 
 	return &copy
 }
 
 // Append string to chain path.
 func (c *chain) enter(name string, args ...interface{}) {
 	c.context.Path = append(c.context.Path, fmt.Sprintf(name, args...))
+	c.context.AliasedPath = append(c.context.AliasedPath, fmt.Sprintf(name, args...))
```

Comment:
In addition to enter(), please also update replace().

Reply:
I fixed on 534ab72 .